src/x86/executor/sandbox_manager.c

```diff
@@ -86,6 +86,22 @@ int allocate_sandbox(void)
     return 0;
 }
 
+/// @brief Returns the number of pages allocated for the sandbox, including util area, code and data
+/// @param void
+/// @return number of pages; -1 on error
+int get_n_sandbox_pages(void) {
```

Comment:
I think the name is a bit misleading (to me it sounds like "allocate n pages for the sandbox"). How about `get_sandbox_size_pages` or something alike?

Reply:
fixed